src/sentry/api/endpoints/organization_member/index.py

```diff
@@ -34,22 +40,42 @@ class MemberConflictValidationError(serializers.ValidationError):
     pass
 
 
-class OrganizationMemberSerializer(serializers.Serializer):
-    email = AllowedEmailField(max_length=75, required=True)
+@extend_schema_serializer(
+    deprecate_fields=["role", "teams"], exclude_fields=["regenerate", "role", "teams"]
+)
+class OrganizationMemberRequestSerializer(serializers.Serializer):
+    email = AllowedEmailField(
+        max_length=75, required=True, help_text="The email address to send the invitation to"
+    )
     role = serializers.ChoiceField(
         choices=roles.get_choices(), default=organization_roles.get_default().id
     )  # deprecated, use orgRole
     orgRole = serializers.ChoiceField(
-        choices=roles.get_choices(), default=organization_roles.get_default().id, required=False
+        choices=roles.get_choices(),
+        default=organization_roles.get_default().id,
+        required=False,
+        help_text="The org-role of the new member",
     )
     teams = serializers.ListField(
         required=False, allow_null=False, default=[]
     )  # deprecated, use teamRoles
     teamRoles = serializers.ListField(
-        required=False, allow_null=True, default=[], child=serializers.JSONField()
+        required=False,
+        allow_null=True,
+        default=[],
+        child=serializers.JSONField(),
+        help_text="The team and team-roles assigned to the member",
+    )
+    sendInvite = serializers.BooleanField(
+        required=False,
+        default=True,
+        write_only=True,
+        help_text="Should send invite notification through email",
```

Comment:
I don't actually remember if the Sentry docs would automatically document this with a note that this defaults to True (if only we had the Vercel docs preview here too so I could easily verify that 👀) but if it doesn't we should explicitly document that